scripts/integration/shutdown/test.yaml

```diff
@@ -6,6 +6,8 @@ args:
 - --test-threads
 - '4'
 
+needs_docker_sock: true
```

Comment:
So I did just think of an alternative - we could just start `dind` as a "regular" service, that might make more sense if it's the only thing that's going to use the docker socket?

Reply:
I like that as a more generic solution. I'll look into it. We don't even need a "real" docker for this, just something that reacts to the socket with some dummy data, so an emulated response code would be even better.

Reply:
Sorry I didn't think of that sooner 😅

Reply:
For posterity: Running DinD, even "rootless" DinD, requires running the service as privileged, which is probably undesirable. This is an easier fix for now.